packages/babel-cli/src/babel/dir.js

```diff
@@ -56,7 +56,7 @@ export default async function({
       outputFileSync(dest, res.code);
       util.chmod(src, dest);
 
-      if (cliOptions.verbose) {
+      if (cliOptions.verbose && !cliOptions.quiet) {
```

Comment:
I'd prefer to throw if both are enabled

Reply:
@nicolo-ribaudo Do you still want me to throw an error here?